plenum/server/node.py

```diff
@@ -3383,11 +3406,13 @@ def onBatchCreated(self, three_pc_batch: ThreePcBatch):
         :return:
         """
         ledger_id = three_pc_batch.ledger_id
-        if ledger_id == POOL_LEDGER_ID:
-            if isinstance(self.poolManager, TxnPoolManager):
-                self.get_req_handler(POOL_LEDGER_ID).onBatchCreated(three_pc_batch.state_root, three_pc_batch.pp_time)
-        elif self.get_req_handler(ledger_id):
-            self.get_req_handler(ledger_id).onBatchCreated(three_pc_batch.state_root, three_pc_batch.pp_time)
+        if self.write_manager.is_valid_ledger_id(ledger_id):
```

Comment:
Why do we need a check for `is_valid_ledger_id` here? Can write_manager just do nothing is unknown ledger_id is passed?